src/main/java/net/azisaba/ryuzupluginchat/redis/MessageSubscriber.java

```diff
@@ -138,6 +157,36 @@ public void onPMessage(String pattern, String channel, String message) {
     }
   }
 
+  public long ping() {
+    if (subscriber == null || !subscriber.isSubscribed()) {
+      return -2;
+    }
+
+    Thread thread =
+        new Thread(
+            () -> {
+              try {
+                Thread.sleep(5000);
+              } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+              }
+            });
+    long start = System.currentTimeMillis();
+
+    pingPongQueue.add(arg -> thread.interrupt());
+    try {
+      subscriber.ping();
+    } catch (JedisConnectionException e) {
+      return -1;
+    }
+
+    try {
+      thread.join(3000);
+    } catch (InterruptedException ignored) {
```

Comment:
InterruptedException should not be ignored

```suggestion
    } catch (InterruptedException ignored) {
        Thread.currentThread().interrupt();
```